apps/vite/src/pages/Home.tsx

```
@@ -1,57 +1,58 @@
-import Spinner from "../components/Spinner"
-import { CalendarData, Months } from "@miti/types"
-import { CalendarEventsResult } from "@miti/types"
-import UpcomingEvents from "./UpcomingEvents"
-import { useYearlyData } from "@miti/query/calendar"
+import { useEffect, useMemo, useState } from 'react';
+import { useQuery } from '@tanstack/react-query';
+import NepaliDate from 'nepali-datetime';
+import { useParams } from 'react-router-dom';
+
+import { useYearlyData } from '@miti/query/calendar';
+import { CalendarData, CalendarEventsResult, Months } from '@miti/types';
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the compilers and avoids loading unnecessary modules.
Safe fix: Add inline type keywords.


(lint/style/useImportType)